### PR TITLE
Varslingsknapp

### PR DESCRIPTION
**Pull request by aslakhol**
This PR creates a button with a simple mailto link to rapport uncomfortable events that occur during fadderuka. This is intended as an MVP to allow us to further develop a better warning system for Online. This should not be considered the last step in our work on this front. 

The PR includes a section close to the bottom of the splash: 
image

The button opens the users preferred email client with some prefilled information:
image

**Things I'd like input on**:

1. The language - The wording might be a bit formal, do we want this? In any case, I'm very open to suggested changes to the wording. 
2. The phone number section - Should we post the numbers of Mathias and Robin? This should not be merged until they have a say in this. 
3. The position of the section - Right now the section is placed towards the bottom, right underneath the slack-invite-section and the SoMe icons. Is this a good placement. 

**Steps before merging**

- [x] Actual code review
- [x] Robin is ok with posting his number
- [x] Mathias is ok with number being posted
- [x] The numbers of Robin and Mathias are added
- [ ] HS approves the language

### CODE REVIEW

**Comment by oleast** [comment]
I think it looks great as an MVP, and having it like this is miles batter than not having it!

For your points:
1. I think it works, and can be used like it is. For how formal to make it I think it will help if the text is also inviting, in a away that makes it clear that you can talk to them in _any_ situation?

2. I think it is important to be able to contact them by phone in case anything happens. Of course they should have a say in it, but I also think that will work out fine. It seems kinda strange to by in their position and not available by phone?

3. I think it works as it is, and is it kinda hard to work out exactly how to place new sections in the app as it gets larger. Maybe it is possible to add another link to the section further up on the page, or add something to make the division between sections clearer?

I also have a little suggestion about further work:

1. What do you think about adding the option to send anonymous messages?

2. Should we (in the future) implement message sending as an integrated part of the app, instead of as a mailto?

Overall it looks good!

**Comment by aslakhol** [comment]
> I think it looks great as an MVP, and having it like this is miles batter than not having it!

Thanks for your review!

> For your points:
> 
> 1. I think it works, and can be used like it is. For how formal to make it I think it will help if the text is also inviting, in a away that makes it clear that you can talk to them in _any_ situation?

I agree, would love suggestions on the wording. 

> 2. I think it is important to be able to contact them by phone in case anything happens. Of course they should have a say in it, but I also think that will work out fine. It seems kinda strange to by in their position and not available by phone?

Both of them have approved this now, so I've just pushed a commit with their actual numbers. 

> 3. I think it works as it is, and is it kinda hard to work out exactly how to place new sections in the app as it gets larger. Maybe it is possible to add another link to the section further up on the page, or add something to make the division between sections clearer?

I agree with what you're saying. I think we should keep it like this for now, but possibly acknowledge that it might be time to revisit our solution for digital fadderuke interface. Maybe we've reached a complexity level where we need another solution than just the simple one-pager?

> I also have a little suggestion about further work:
> 
> 1. What do you think about adding the option to send anonymous messages?

I considered this, but looking at different #metoo buttons and the like from other institutions almost all that allowed anonymous messages made it clear that messages without contact info would most likely not lead to any sanctions or changes, as the message would be too hard to verify or follow up. I do think this is something that we should include if we do the work I describe in the answer to your next suggestion. 

> 2. Should we (in the future) implement message sending as an integrated part of the app, instead of as a mailto?

I totally agree that we need to have a better solution than this in the long term, however, I don't think it should be a part of the splash. In my opinion, we should have a system for handling these sorts of cases through all of Online, and that system should include a digital interface for sending warnings. I think the interface should live either in OWF or in a separate app (and obviously we should link it on the splash and in other places). 


> Overall it looks good!
Thanks!

**Comment by mathisfo** [comment]
Outstand MVP!

> The language - The wording might be a bit formal, do we want this? In any case, I'm very open to suggested changes to the wording.

The Board will get back to you with text details. I dont think its too formal.

> The phone number section - Should we post the numbers of Mathias and Robin? This should not be merged until they have a say in this.

All good

> The position of the section - Right now the section is placed towards the bottom, right underneath the slack-invite-section and the SoMe icons. Is this a good placement.

Feedback from a meeting with the board members involved suggests that this section should be moved further up (IIRC the splash is quite long). I think it would be wrong to put it at the top. Some balance would be nice.

To follow up on @oleast comments:
> 1. What do you think about adding the option to send anonymous messages?

My personal preference is that anonymity is an option. I would be more than happy to supply a wording and a course of action for forms sent anonymously.

> 2. Should we (in the future) implement message sending as an integrated part of the app, instead of as a mailto?

 A "task force" is soon to be established on working for sustainable handling of whistleblower cases in Online. This question is most likely fully answered in the coming months.

**Comment by aslakhol** [comment]
@mathisfo How far up do you think the section should be moved? I landed on this placing because I didn't think any was better but I'm totally willing to hear arguments for any other specific placement. This is the current order:
image


I also agree it's getting a bit long. It might be closing in on the time to reconsider our fadderuke surface, but then again it's not our most pressing concern. 

Also bump on the wording.

**Comment by oleast** [comment]
I think that the page works mostly as it is even though it contains a lot of information. Maybe adding some anchor links or a list of content at the top that you can click on to help the user along?
Maybe some longer texts could be hidden unless something is interacted with, and the content is a bit more bordered or distanced from each other?

TLDR: I think some small design tweaks goes a long way in making the page more navigable at a glance
